.gitignore

```diff
@@ -1,3 +1,4 @@
 /node_modules/
 /.idea/
 /.next/
+.env*
```

Comment:
graag ook toevoegen _wat_ er dan in je .env.whatever moet staan, in de readme bijvoorbeeld